doc/installing.rst

```diff
@@ -30,10 +30,10 @@ System Requirements
 - Windows 7+
 - Mac OS X 10.7+ (**64-bit only**)
 - CentOS 6 & 7 (64-bit only)
-- Debian 7.0 & 8.0 & 9.0
-- Fedora 24 & 25 & 26
-- Ubuntu 16.04 & 16.10 & 17.04
-- openSUSE Leap 42.1 & 42.2 & 42.3
+- Debian 8.0 & 9.0
+- Fedora 25 & 26 & 27
+- Ubuntu 16.04 & 16.10 & 17.10
```

Comment:
so 17.04 is no longer supported while 16.10 still is?  That's a bit strange.

Reply:
That seems wrong. Good catch.
The last line should be Ubuntu 16.04 & 17.04 & 17.10 unless @michaelstingl knows better.